modules/tf_cloudbuild_builder/variables.tf

```diff
@@ -70,7 +70,7 @@ variable "gar_repo_name" {
 variable "gar_repo_location" {
   description = "Name of the location for the Google Artifact Repository."
   type        = string
-  default     = "us"
+  default     = "us-central1"
```

Comment:
I think it's probably better to remove the default entirely, rather than change the default value.  That way during an module upgrade an error is surfaced if using the default, rather than the value unexpectedly changing.